torch/distributed/nn/jit/instantiator.py

```diff
@@ -19,6 +15,10 @@
 
 
 _FILE_PREFIX = "_remote_module_"
+_TEMP_DIR = tempfile.TemporaryDirectory()
```

Comment:
Would I be correct if  assume each process will have its own temp dir to write/read generated code?

Reply:
@mrshenli Yes, each process / Python Interpreter / Worker will have its own temp dir.